src/app/backend-api.service.ts

```diff
@@ -3,11 +3,18 @@ import {HttpClient} from '@angular/common/http';
 import {Observable} from 'rxjs';
 import {environment} from '../environments/environment';
 
+interface UserProfile {
+  username: string;
+  profilePic: any;
+}
+
 @Injectable({
   providedIn: 'root'
 })
+
 export class BackendAPIService {
   endpoint = `https://${environment.nodeHostname}/api/v0`;
+  exchange = `https://${environment.nodeHostname}/api/v1`;
```

Comment:
This endpoint is way too slow for just fetching the tip height. We probably need a new endpoint. Or, alternatively, Lazy Nina just merged a PR that adds GetAppState to this file. Let's include the tip height in the app state.